linode/manager
# Upgrade Cypress to v9.5.1

## Description

**What does this PR do?**
This upgrades Cypress from 8.6.x to 9.5.1, and updates related dependencies as well.

I was hoping this might be low hanging fruit to improve the performance of our tests, but sadly performance seems to be about the same with these changes in place.

## How to test

**What are the steps to reproduce the issue or verify the changes?**
Run `yarn up` to ensure that packages are upgraded, and in a separate console run `yarn cy:run` and confirm that no failures occur. This takes between 20-25 minutes on my machine.
